web/themes/custom/droopler_theme/src/components/blog/blog-teaser/blog-teaser.twig

```diff
@@ -35,7 +35,7 @@
       </a>
     </div>
 
-    <div {{ bem('content', [], base_class, ['col-md-5', 'col-xl-5', 'offset-xl-1']) }}>
+    <div {{ bem('content', [''], base_class, ['col-md-5', 'col-xl-5', 'offset-xl-1']) }}>
```

Comment:
do we need that change?

Reply:
This is needed so that the section padding does not duplicate itself in the block.